addon/components/base/bs-form.js

```diff
@@ -136,6 +137,31 @@ export default Component.extend({
    */
   horizontalLabelGridClass: 'col-md-4',
 
+  /**
+   * `isSubmitting` is `true` after `submit` event has been triggered and until Promise returned by `onSubmit` is
+   * fulfilled. If `validate` returns a Promise that one is also taken into consideration.
+   *
+   * `isSubmitting` property is yielded and could be used to customize form submit button. E.g. a loading spinner could
+   * be shown and the submit button been disabled:
+   *
+   * ```
+   * {{#bs-form onSubmit=(action 'save') as |form|}}
+   *   {{#bs-button type='submit' disabled=form.isSubmitting}}
+   *     Save {{fa-icon 'spinner'}}
+   *   {{/bs-button}}
+   * {{/bs-from}}
+   * ```
+   *
+   * The value is not accurate if multiple submit events are pending. In that case `isSubmitting` is `false` as soon
+   * as the first submission fulfills.
```

Comment:
If we like to support that use case, this limitation could also be dropped. I think adding a counter of submissions and only setting `isSubmitting` to `false` for the last pending one should be enough.

Reply:
I'm not convinced of adding `ember-concurrency` as dependency to be honest. It's a great addon but is it really needed? In app code it's great to keep things simple but it's a heavy dependency for addons and couples libraries with different scope. Also makes internals harder to understand for users and contributors not used to `ember-concurrency`.

Also I'm not quite sure if we should force the submit to be not fired again before the last submission is fulfilled. It would match my current use cases but there may be others. So it should be at least configurable. I'm willing to add that as a configurable option in another PR. It's basically just checking `isSubmitting === true`.